config/environment.js

```diff
@@ -117,6 +117,24 @@ module.exports = function (environment) {
         }
       }
     },
+
+    metricsAdapters: [
+      {
+        name: 'GoogleAnalytics',
+        environments: ['development', 'production'],
+        config: {
+          id: 'UA-XXXX-Y',
```

Comment:
I suggest to use value from config here
```suggestion
          id: process.env.GOOGLE_ANALYTICS_KEY,
```

Reply:
Cool, thank you @emmanuel-ci !
Great work 🎉 👏 👍